managedkafka/examples/src/main/java/examples/CreateCluster.java

```diff
@@ -64,17 +72,59 @@ public static void createCluster(
             .setRebalanceConfig(rebalanceConfig)
             .build();
 
-    try (ManagedKafkaClient managedKafkaClient = ManagedKafkaClient.create()) {
+    // Create the settings to configure the timeout for polling operations
+    ManagedKafkaSettings.Builder settingsBuilder = ManagedKafkaSettings.newBuilder();
+    TimedRetryAlgorithm timedRetryAlgorithm = OperationTimedPollAlgorithm.create(
+        RetrySettings.newBuilder()
+            .setInitialRetryDelayDuration(Duration.ofSeconds(5L))
+            .setRetryDelayMultiplier(1.5)
+            .setMaxRetryDelayDuration(Duration.ofSeconds(30L))
+            .setInitialRpcTimeoutDuration(Duration.ZERO) // ignored
+            .setRpcTimeoutMultiplier(1.0) // ignored
+            .setMaxRpcTimeoutDuration(Duration.ZERO) // ignored
+            .setTotalTimeoutDuration(Duration.ofHours(1L)) // set polling timeout to 1 hour
+            .build());
+    settingsBuilder.createClusterOperationSettings()
+        .setPollingAlgorithm(timedRetryAlgorithm);
+
+    try (ManagedKafkaClient managedKafkaClient = ManagedKafkaClient.create(
+        settingsBuilder.build())) {
+      // Optionally print OperationCallSettings to investigate LRO timeout errors
+      // due to configuration set on the client side
+      // System.out.println("Create Cluster Settings (OperationCallSettings): " +
+      //     managedKafkaClient.getSettings().createClusterOperationSettings().getPollingAlgorithm()
+      //         .toString());
+
       CreateClusterRequest request =
           CreateClusterRequest.newBuilder()
               .setParent(LocationName.of(projectId, region).toString())
               .setClusterId(clusterId)
               .setCluster(cluster)
               .build();
+
       // The duration of this operation can vary considerably, typically taking between 10-40
       // minutes.
       OperationFuture<Cluster, OperationMetadata> future =
           managedKafkaClient.createClusterOperationCallable().futureCall(request);
+
+      // Get the initial LRO and print details.
+      OperationSnapshot operation = future.getInitialFuture().get();
+      System.out.printf("Cluster creation started. Operation name: %s\nDone: %s\nMetadata: %s\n",
+          operation.getName(),
+          operation.isDone(),
+          future.getMetadata().get().toString());
+
+      // Polling logs to see operation progress
```

Comment:
I don't think we need this comment, and it's confusing (should be just Polling for operation progress).

Reply:
Removed!

The reason I said "polling logs .." is because future.get() also polls under the hood so this while loop is doing the same thing but so we can see progress by looking at the log statements being printed out.